src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -369,29 +370,13 @@ public static boolean isGenomicsDBPath(final String path) {
             throw new IllegalArgumentException("Trying to create a GenomicsDBReader from a non-GenomicsDB input");
         }
 
```

Comment:
Instead of `gendb://gs://` as the URI convention, can you introduce a new URI scheme for "gendb on gcs"? Eg., `gendbgs://` (and still also accept `gendb://` for local gendb instances). 

This way, the URIs will at least be legal...

Reply:
For the URI schemes, would you consider gendb://, gendb.gs:// gendb.hdfs://, etc.?
According to [wikipedia](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier), these are valid schemes - A non-empty scheme component followed by a colon (:), consisting of a sequence of characters beginning with a letter and followed by any combination of letters, digits, plus (+), period (.), or hyphen (-).

Reply:
If a period in the scheme is legal, then these would be fine. I'd check to make sure that Java's URI, URL, and `java.nio.file.Path` classes accept such schemes, though.

Reply:
Yes, URI and Path recognize period in their schemes.